qiskit_nature/problems/sampling/protein_folding/data_loaders/energy_matrix_loader.py

```diff
@@ -0,0 +1,37 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""Loads the energy matrix from the Miyazawa-Jernigan potential file."""
+from typing import Tuple, List
+
+import numpy as np
+
+
+def _load_energy_matrix_file() -> Tuple[np.ndarray, List[str]]:
+    """Returns the energy matrix from the Miyazawa-Jernigan potential file."""
+
+    path = "qiskit_nature/problems/sampling/protein_folding/resources/mj_matrix.txt"
```

Comment:
Is this the only place which couples to this `mj_matrix`? If so, I would argue you could move it into this module and document what that file contains as part of this modules `__init__` file.

Reply:
Yes, but in the future there are plans to have more files like this, also user-defined, so it might make sense to have such a designated place to store them.